src/main/java/gov/usgs/cida/pubs/domain/PersonContributor.java

```diff
@@ -24,6 +27,9 @@
 @Component
 @ParentExists
 public class PersonContributor<D> extends Contributor<PersonContributor<D>> implements ILookup {
+	public static final String ORCID_VALIDATION_REGEX = ".*" + DataNormalizationUtils.ORCID_REGEX;
```

Comment:
Now that just the 19 characters are being stored, can the ORCID_VALIDATION_REGEX and the DataNormalizationUtils.ORCID_REGEX be the same value?

Reply:
Good catch, was thinking about that this weekend. :-)